### ISSUE TITLE
RFE: Can kadalu operator take external gluster server and manage only CSI?

### ISSUE THREAD

**Issue by amarts** [kadalu/kadalu#36]
There are many possible setup where Gluster servers can be outside the k8s cluster, and there may be need for just providing CSI driver in k8s. Can the operator handle it?

I am thinking of config file to say something like:

```
# File: storage-config.yaml
---
apiVersion: kadalu-operator.storage/v1alpha1
kind: KadaluStorage
metadata:
  # This will be used as name of PV Hosting Volume
  name: storage-pool-1
spec:
  type: External   #Here, volume type can be anything
  storage:
    - node: node1
      volume: volume_name # Volume name
      options:
          - backup_volfile_server: node2, node3
             log_level: DEBUG
```

We can recommend to run the quota setting script as hook script in servers for such users, as we can't control setting of quota.

**Comment by aravindavk** [comment]
This is a very good enhancement!  +1

**Comment by holmesb** [comment]
+1, this would probably unlock our ability to use this project.  Combining our storage and compute (k8s) nodes is too limiting.

**Comment by amarts** [comment]
This is now functional and merged to master branch, should be available with v0.4
